python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -18,99 +17,54 @@
+        ai_model_id: str | None = None,
+        service_id: str | None = None,
+        api_key: str | None = None,
+        org_id: str | None = None,
+        default_headers: Mapping[str, str] | None = None,
+        async_client: AsyncOpenAI | None = None,
+        env_file_path: str | None = None,
     ) -> None:
         """
         Initialize an OpenAIChatCompletion service.
 
         Arguments:
             ai_model_id {str} -- OpenAI model name, see
                 https://platform.openai.com/docs/models
-            async_client {AsyncOpenAI} -- An existing client to use.
-        """
-
-    @overload
-    def __init__(
-        self,
-        ai_model_id: str,
-        api_key: Optional[str] = None,
-        org_id: Optional[str] = None,
-        service_id: Optional[str] = None,
-        default_headers: Optional[Mapping[str, str]] = None,
-    ) -> None:
-        """
-        Initialize an OpenAIChatCompletion service.
-
-        Arguments:
-            ai_model_id {str} -- OpenAI model name, see
-                https://platform.openai.com/docs/models
-            api_key {Optional[str]} -- OpenAI API key, see
-                https://platform.openai.com/account/api-keys
-            org_id {Optional[str]} -- OpenAI organization ID.
-                This is usually optional unless your
-                account belongs to multiple organizations.
-            default_headers: The default headers mapping of string keys to
-                string values for HTTP requests. (Optional)
-        """
-
-    @overload
-    def __init__(
-        self,
-        ai_model_id: str,
-        api_key: Optional[str] = None,
-        service_id: Optional[str] = None,
-        default_headers: Optional[Mapping[str, str]] = None,
-    ) -> None:
-        """
-        Initialize an OpenAIChatCompletion service.
-
-        Arguments:
-            ai_model_id {str} -- OpenAI model name, see
-                https://platform.openai.com/docs/models
-            api_key {Optional[str]} -- OpenAI API key, see
-                https://platform.openai.com/account/api-keys
+            service_id {str | None} -- Service ID tied to the execution settings.
+            api_key {str | None} -- The optional API key to use. If provided will override,
+                the env vars or .env file value.
+            org_id {str | None} -- The optional org ID to use. If provided will override,
+                the env vars or .env file value.
             default_headers: The default headers mapping of string keys to
                 string values for HTTP requests. (Optional)
+            async_client {Optional[AsyncOpenAI]} -- An existing client to use. (Optional)
+            env_file_path {str | None} -- Use the environment settings file as a fallback
+                to environment variables. (Optional)
         """
+        try:
+            openai_settings = OpenAISettings.create(env_file_path=env_file_path)
+        except ValidationError as e:
```

Comment:
This should prob not happen, you can supply the api key directly, so should just move along and then it will be caught later

Reply:
As we talked about offline, it's not terrible to leave in the exception as the ValidationError is raised only if env vars are found that match the prefix. The validation error should get handled later on when validating the final pydantic model, so will remove the exception for now, and change the logger error to a warning.